components/pokemonsplashart.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version = "1.0" encoding = "utf-8" ?>
+<component name="PokemonSplashArt" extends="Group">
```

Comment:
Can you please rename this component?
It is not really a splash art:
``` Splash art is a modification of a splash screen. A splash screen used to be (well, still is in some cases) a graphic along with a logo and sometimes the current version of the software involved. It was generally the screen that would be displayed while the software was loading when it first started up. ```
The component can contain more than just a poster. Consider some of the options: "backgroundScreen","posterScreen" or "pokeArtScreen"